ansible/roles/providers/tasks/aws.yml

```diff
@@ -33,6 +33,36 @@
   when:
     - ansible_distribution == "Flatcar"
 
+
+- name: install cloud-init deps
```

Comment:
Curious why this task is needed and if they are deps why rpm doesn't install it

Reply:
i broke these apart so cloud-init can be installed separately

Reply:
Sorry I still don't understand, are these not defined as dependencies of `cloud-init` in the package definition? Is that why you need to explicitly install it here?

Reply:
cloud-init is installed with `allow_downgrade` they were previously all installed in one block as they are in vmware

```

- name: Install cloud-init packages
  yum:
    name: "{{ packages }}"
    state: present
    enablerepo: "{{ 'offline' if offline_mode_enabled else '' }}"
    disablerepo: "{{ '*' if offline_mode_enabled else '' }}"
  vars:
    packages:
      - cloud-init
      - cloud-utils-growpart
      - "python{{ '3' if ansible_distribution_major_version|int >= 9 else '2'}}-pip"
  ```
  
  we want the last two without allow_downgrade